### ISSUE TITLE
Build debug bundle using --debug

### ISSUE THREAD

**Issue by lauff** [abernix/meteord#29]
Hi & Happy New Year!

Is it possible to build a bundle with the --debug parameter?

Thanks
Markus

**Comment by abernix** [comment]
The `--debug` flag is only relevant during development with the `meteor` command.  The built bundle (for production, which is a Node.js app, and no longer aware of the `meteor` command), doesn't support `--debug`.

**Comment by abernix** [comment]
Sorry, I may have closed this prematurely, and it seems that `--debug` on `meteor build` probably does something different than what I was thinking you were trying to do.

Are you trying to skip the minifying step by using the `--debug` flag?

**Comment by lauff** [comment]
yes, I use the flag to skip the minifying, but also to load dev dependencies (e.g. Meteor Toys).
